src/components/menus/Items.ts

```diff
@@ -114,6 +115,29 @@ export class Items<TGameStartr extends FullScreenPokemon> extends Component<TGam
         const options: any[] = [
             {
                 callback: (): void => {
+                    const category = this.gameStarter.itemsHolder.getItem(listing.item).category;
```

Comment:
I still need to get you that API for knowing if we're in Gen II mode, let me get on that...